server/osint/tools/lib/tool_maigret.py

```diff
@@ -0,0 +1,51 @@
+import os
+import json
+import subprocess
+
+
+def run_maigret(run):
+    # print("Current working directory:", os.getcwd())
+    try:
+        subprocess.Popen(['maigret', run.input_value, '--json', 'simple'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        run.status = 'running'
+    except Exception as e:
+        message = f'Run maigret failed. Username is {run.input_value}. Return code: {e}'
+        run.status = 'error'
+        return message
+
+    run.save()
+    return run.run_id
+
+
+def check_maigret(run):
+    if run.status == 'initiate':
```

Comment:
- Is **initiate** in run.status default value?

Reply:
before: running -> completed / error
after: initiate -> running -> completed / error